dokka-subprojects/plugin-javadoc/src/main/kotlin/org/jetbrains/dokka/javadoc/JavadocPlugin.kt

```diff
@@ -27,10 +27,11 @@ import org.jetbrains.dokka.transformers.documentation.PreMergeDocumentableTransf
 import org.jetbrains.dokka.transformers.pages.PageTransformer
 import org.jetbrains.dokka.validity.PreGenerationChecker
 
-public class JavadocPlugin : DokkaPlugin() {
+public class JavadocPlugin : DokkaPlugin(), WithUnsafeExtensionSuppression {
 
     private val dokkaBasePlugin: DokkaBase by lazy { plugin<DokkaBase>() }
     private val kotinAsJavaPlugin: KotlinAsJavaPlugin by lazy { plugin<KotlinAsJavaPlugin>() }
+    public override val extensionsSuppressed: List<Extension<*, *, *>> by lazy { listOf(dokkaBasePlugin.defaultSamplesTransformer) }
```

Comment:
or 
```kt
    public val emptySampleTransformer: Extension<PageTransformer, *, *> by extending {
        CoreExtensions.pageTransformer providing {
            PageTransformer { it }
        } override dokkaBasePlugin.defaultSamplesTransformer
    }
````

Reply:
I personally like the proposed version with `override` more, I think it's simpler

Before today, I didn't even know `WithUnsafeExtensionSuppression` existed :open_mouth: